src/bb_excel/core.clj

```diff
@@ -132,68 +133,74 @@
 
 (defn process-cell
   "Process Excel cell"
-  [dict styles coll]
-  (let [[[_ row col]] (re-seq #"([A-Z]*)([0-9]+)" (:r coll))
-        u (-> coll
-              (assoc :x row)
-              (assoc :y col))]
+  [dict styles cell]
+  (let [[_ row-index col-index] (re-matches #"([A-Z]+)([0-9]+)" (:r cell))
+        cell* (merge cell
+                     {:x row-index
+                      :y col-index})
+        cell-type (:t cell*)
+        cell-value (:d cell*)]
     (cond
-      ;; Possible data types well explained here https://stackoverflow.com/a/18346273
-      (= (:t u) "s")                (dissoc (assoc-in u [:d] (dict (read-string (:d u)))) :t)
-      (= (:t u) "str")              (dissoc u :t)
-      (= (:t u) "inlineStr")        (dissoc u :t)
```

Comment:
don't understand why we need dissoc :t, we completely transform this data in next step

Reply:
Can't remember why either. Ok to remove.